pkg/kubelet/container_gc.go

```diff
@@ -157,6 +170,18 @@ func (cgc *realContainerGC) GarbageCollect() error {
 		}
 	}
 
+	// Remove dead symlinks - should only happen on upgrade
+	// from a k8s version without proper log symlink cleanup
+	logSymlinks, _ := filepath.Glob(path.Join(cgc.containerLogsDir, "*.log"))
```

Comment:
"*.log" is fragile here, as there's nothing explicit keeping it in sync with the log file names/symlinks themselves.  Best make the file extension a shared constant to keep the log file names and this wildcard in sync, or else elevate this literal string to a named constant at the top of the file, with reciprocal comments to keep it in sync with the log file names.